gax/src/main/java/com/google/api/gax/rpc/BatchingCallSettings.java

```diff
@@ -40,7 +41,8 @@
  * A settings class to configure a {@link UnaryCallable} for calls to an API method that supports
  * batching. The settings are provided using an instance of {@link BatchingSettings}.
  */
-@BetaApi
+@BetaApi("The surface for batching is not stable yet and may change in the future.")
+@InternalExtensionOnly
```

Comment:
What does it mean for a class to be both `@InternalExtensionOnly` and `final`?

Reply:
It's redundant honestly, and just insurance for if someone decides to remove the `final` later.

Reply:
Ok - if we are happy to keep the redundancy then that is fine.